Tests fail on python 3.8

```
##[error]1 test(s) failed, 0 test(s) collected.
Skipping uploading of coverage data.

==================================== ERRORS ====================================
__________________ ERROR collecting tests/test_enochecker.py ___________________
tests/test_enochecker.py:12: in <module>
    class CheckerExampleImpl(BaseChecker):
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/enochecker/enochecker.py:198: in __init__
    cls.service = init_service(cls)  # type: Flask
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/enochecker/checkerservice.py:333: in init_service
    app = Flask(__name__)
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/flask/app.py:558: in __init__
    self.add_url_rule(
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/flask/app.py:66: in wrapper_func
    return f(self, *args, **kwargs)
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/flask/app.py:1216: in add_url_rule
    self.url_map.add(rule)
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/werkzeug/routing.py:1386: in add
    rule.bind(self)
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/werkzeug/routing.py:730: in bind
    self.compile()
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/werkzeug/routing.py:794: in compile
    self._build = self._compile_builder(False).__get__(self, None)
/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/werkzeug/routing.py:945: in _compile_builder
    code = compile(module, "<werkzeug routing>", "exec")
E   TypeError: required field "type_ignores" missing from Module
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -19,6 +19,8 @@ pool:
   vmImage: 'ubuntu-latest'
 strategy:
   matrix:
+    Python35:
+      python.version: '3.5'
     Python36:
       python.version: '3.6'
     Python37:

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -23,6 +23,8 @@ strategy:
       python.version: '3.6'
     Python37:
       python.version: '3.7'
+    Python38:
+      python.version: '3.8'
 steps:
 - task: UsePythonVersion@0
   inputs:

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -19,8 +19,6 @@ pool:
   vmImage: 'ubuntu-latest'
 strategy:
   matrix:
-    Python35:
-      python.version: '3.5'
     Python36:
       python.version: '3.6'
     Python37:

--- a/requirements.txt
+++ b/requirements.txt
@@ -6,12 +6,12 @@ chardet==3.0.4
 Click==7.0
 colorama==0.4.1
 elastic-apm==4.2.2
-Flask==1.0.2
+Flask==1.0.3
 future==0.17.1
 futures==3.1.1
 idna==2.8
 itsdangerous==1.1.0
-Jinja2==2.10.1
+Jinja2==2.10.2
 MarkupSafe==1.1.1
 pathlib2==2.3.3
 pluggy==0.9.0
@@ -25,4 +25,4 @@ six==1.12.0
 typing==3.6.6
 urllib3==1.24.3
 websocket-client==0.56.0
-Werkzeug==0.15.3
+Werkzeug==0.15.5

--- a/src/enochecker/nosqldict.py
+++ b/src/enochecker/nosqldict.py
@@ -1,4 +1,4 @@
-import collections
+from collections.abc import MutableMapping
 import logging
 import os
 from functools import wraps
@@ -46,7 +46,7 @@ def _try_n_times(func):
     return try_n_times
 
 
-class NoSqlDict(collections.MutableMapping):
+class NoSqlDict(MutableMapping):
     """
     A dictionary that is MongoDb backed.
     """

--- a/src/enochecker/storeddict.py
+++ b/src/enochecker/storeddict.py
@@ -3,7 +3,7 @@ from future.standard_library import install_aliases
 install_aliases()
 
 import atexit
-import collections
+from collections.abc import MutableMapping
 import json
 import os
 import threading
@@ -72,7 +72,7 @@ def _locked(func):
     return cast(T, locked)
 
 
-class StoredDict(collections.MutableMapping):
+class StoredDict(MutableMapping):
     """
     A dictionary that is filesystem backed.
     It will write to disk every few seconds and at exit.

